### PR TITLE
adds new themes to the bani overlay

### PR DESCRIPTION
**Pull request by inderpreetsingh**
https://www.youtube.com/watch?v=SP5kblnGWeY&feature=youtu.be

Adds two themes
- Classic 
- Vertical

### CODE REVIEW

File: www/obs/index.html [hunk 1]
```diff
@@ -12,9 +12,7 @@
         Gurmukhi: 'ik®pw kry jy AwpxI qw hir rKw aur Dwir ]1]',
         English: 'If the Lord shows His Mercy, then I keep Him enshrined within my heart. ||1||',
         Punjabi: 'jy prmwqmw AwpxI myhr kry, qW mYN BI aus dw nwm ihrdy ivc pro r`KW [1[',
-        Transliteration: {
-          English: 'kirapaa kare je aapanee taa har rakhaa aur dhaar ||1||',
-        },
+        Transliteration: 'kirapaa kare je aapanee taa har rakhaa aur dhaar ||1||',
```
Comment (tsingh777) [inline www/obs/index.html]: is this how actual lines are sent? does this allow for different translits in the future?

File: www/obs/index.html [hunk 2]
```diff
@@ -71,12 +68,36 @@
...
+          'layout-classic',
+          'layout-vertical',
+          'layout-vertical-left',
+        );
 
         document.body.classList.add('layout-' + overlayPrefs.overlayVars.layout);
 
         document.body.classList.toggle('logo-off', !overlayPrefs.overlayVars.overlayLogo);
 
+        const overrideTheme = function(selector) {
```
[the remark this answers is not in the document]
Comment (tsingh777) [reply www/obs/index.html]: are we eventually going to allow the other settings (opacity, size, color) apply to the new formats?

File: www/obs/index.html [hunk 3]
```diff
@@ -71,12 +68,36 @@
...
+              .forEach(el => {
+                el.style.backgroundColor = 'transparent';
+                el.style.color = '#fff';
+                document.querySelector('.o-gurbani').style.color = '#fff';
+              });
+          }
+        };
+
+        overrideTheme('.layout-classic');
+        overrideTheme('.layout-vertical');
```
[the remark this answers is not in the document]
Comment (inderpreetsingh) [reply www/obs/index.html]: Yeah isn't that the point of these theme + layouts? I thought it has to be certain particular color. @maneetpaul

File: www/js/overlay.js [hunk 4]
```diff
@@ -290,6 +296,8 @@ controlPanel.append(toggleLarivaar);
 const topLayoutBtn = layoutButtonFactory('top');
 const bottomLayoutBtn = layoutButtonFactory('bottom');
 const splitLayoutBtn = layoutButtonFactory('split');
+const verticalLayoutBtn = layoutButtonFactory('vertical');
```
[the remark this answers is not in the document]
Comment (inderpreetsingh) [reply www/js/overlay.js]: Ohh yeah it was in my mind and i totally forgot about it. I will add that icon flip too.

File: www/js/overlay.js [hunk 5]
```diff
@@ -132,12 +132,18 @@ const layoutButtonFactory = layoutName =>
     h('div.layout-bar.layout-bar-2'),
     h('div.layout-bar.layout-bar-3'),
     h('div.layout-bar.layout-bar-4'),
+    h('div.layout-vertical-bar'),
+    h('div.layout-classic-bar'),
```
[the remark this answers is not in the document]
Comment (inderpreetsingh) [reply www/js/overlay.js]: aah! I see.

File: www/obs/index.html (hunk 1 above)
Comment (inderpreetsingh) [reply www/obs/index.html]: I will check andn have it mimic the actual object.

File: www/obs/index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (maneetpaul) [reply www/obs/index.html]: Background color should still selectable, just like the other layouts. Not making any changes to the themes

File: www/js/overlay.js [hunk 6]
```diff
@@ -314,7 +326,16 @@ textControls.append(separatorY());
...
+textControls.append(
+  controlsFactory(
+    [topLayoutBtn, bottomLayoutBtn, splitLayoutBtn, verticalLayoutBtn, classicLayoutBtn],
+    'Layout',
+  ),
+);
+
+document
+  .querySelector('.layout-btn.vertical')
+  .classList.toggle('vertical-left', overlayVars.layout === 'vertical-left');
```
Comment (tsingh777) [inline www/js/overlay.js]: think this needs to be flipped. The icon should be for the opposite of what is being shown so it indicates there another option.

**Pull request merged**